homeassistant/components/sensor/buienradar.py

```diff
@@ -38,6 +40,10 @@
 # Key: ['label', unit, icon]
 SENSOR_TYPES = {
```

Comment:
That will requires quite a bit of work; all functionality should be moved over to the weather component (and all current/future values must be exposed by the weather component)....
Next to that, templating the weather component is not as user-friendly as configuring a sensor....